backend/test_observer/controllers/test_executions/test_executions.py

```diff
@@ -160,3 +161,22 @@ def patch_test_execution(
         test_execution.status = request.status
 
     db.commit()
+
+
+@router.patch("/{id}/review")
+def review_test_execution(
+    id: int,
+    request: TestExecutionsReviewPatchRequest,
+    db: Session = Depends(get_db),
+):
```

Comment:
why not extend regular `PATCH /test-executions/:id` endpoint?

Reply:
I have switched to using this endpoint now. My idea was to have a separate endpoint for this to make the other one cleaner, but I guess it doesn't make sense to add a new endpoint for something that we already have.